internal/bull/notfound.go

```diff
@@ -40,7 +40,10 @@ bull works with pages, so maybe you would like to:
 func (b *bullServer) renderNotFound(w http.ResponseWriter, r *http.Request) error {
 	pageName := pageFromURL(r)
 	var buf bytes.Buffer
-	fmt.Fprintf(&buf, "# Error: page %q not found\n", pageName)
+	fmt.Fprintf(&buf, `# Error: page %q not found
+
+To create this page <a href="%s/edit/%s">click here</a> or press <kbd>Ctrl/Meta<kbd> + <kbd>E</kbd>.
```

Comment:
Please change `%s/edit/%s` to `%sedit/%s`, as the URLBullPrefix() already ends in a / (the double slashes can work, which is probably why you didn’t notice, but they do not necessarily work in all setups).

Reply:
Thank you, fixed! I am letting the browsers work around this kind of thing in the Agora in a few places I'm sure... :)